### ISSUE TITLE
Setting the returnKeyType is making the app crash for iOS < 7.1

### ISSUE THREAD

**Issue by diegoreymendez** [wordpress-mobile/WordPress-iOS#4144]
Log:

```
Thread : Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x2ed85e83 __exceptionPreprocess
1  libobjc.A.dylib                0x390e26c7 objc_exception_throw
2  CoreFoundation                 0x2ed897b7 -[NSObject(NSObject) doesNotRecognizeSelector:]
3  CoreFoundation                 0x2ed880af ___forwarding___
4  CoreFoundation                 0x2ecd6dc8 _CF_forwarding_prep_0
5  WordPress                      0x0015de17 -[WPSearchControllerConfigurator configureSearchBar] (WPSearchControllerConfigurator.m:57)
6  WordPress                      0x0015db3d -[WPSearchControllerConfigurator configureSearchControllerAndWrapperView] (WPSearchControllerConfigurator.m:35)
7  WordPress                      0x000c9a79 -[AbstractPostListViewController configureSearchController] (AbstractPostListViewController.m:230)
8  WordPress                      0x000c8bc1 -[AbstractPostListViewController viewDidLoad] (AbstractPostListViewController.m:39)
9  WordPress                      0x001b011f -[PostListViewController viewDidLoad] (PostListViewController.m:102)
10 UIKit                          0x3150e95b -[UIViewController loadViewIfRequired]
11 UIKit                          0x3150e719 -[UIViewController view]
12 UIKit                          0x3169ac2b -[UINavigationController _startCustomTransition:]
13 UIKit                          0x315b8a17 -[UINavigationController _startDeferredTransitionIfNeeded:]
14 UIKit                          0x315b8821 -[UINavigationController __viewWillLayoutSubviews]
15 UIKit                          0x315b87b9 -[UILayoutContainerView layoutSubviews]
16 UIKit                          0x3150a353 -[UIView(CALayerDelegate) layoutSublayersOfLayer:]
17 QuartzCore                     0x31190943 -[CALayer layoutSublayers]
18 QuartzCore                     0x3118c167 CA::Layer::layout_if_needed(CA::Transaction*)
19 QuartzCore                     0x3118bff9 CA::Layer::layout_and_display_if_needed(CA::Transaction*)
20 QuartzCore                     0x3118ba0d CA::Context::commit_transaction(CA::Transaction*)
21 QuartzCore                     0x3118b81f CA::Transaction::commit()
22 QuartzCore                     0x3118554d CA::Transaction::observer_callback(__CFRunLoopObserver*, unsigned long, void*)
23 CoreFoundation                 0x2ed50f69 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__
24 CoreFoundation                 0x2ed4e8f7 __CFRunLoopDoObservers
25 CoreFoundation                 0x2ed4ec43 __CFRunLoopRun
26 CoreFoundation                 0x2ecb9471 CFRunLoopRunSpecific
27 CoreFoundation                 0x2ecb9253 CFRunLoopRunInMode
28 GraphicsServices               0x339f32eb GSEventRunModal
29 UIKit                          0x3156e845 UIApplicationMain
30 WordPress                      0x00054b3b main (main.m:5)
31 libdyld.dylib                  0x395dbab7 start
```

**Comment by diegoreymendez** [comment]
[Shared the answer](http://stackoverflow.com/questions/31996624/unrecognized-selector-when-calling-uisearchbar-setreturnkeytype/31996625#31996625) of what was causing this issue for us.

**Comment by diegoreymendez** [comment]
Will be released in 5.4.1.
